internal/cmd/controller/gitops/reconciler/gitjob_controller.go

```diff
@@ -101,9 +101,11 @@ func (r *GitJobReconciler) SetupWithManager(mgr ctrl.Manager) error {
 					predicate.GenerationChangedPredicate{},
 					predicate.AnnotationChangedPredicate{},
 					predicate.LabelChangedPredicate{},
+					webhookCommitChangedPredicate(),
 				),
 			),
 		).
+		Owns(&batchv1.Job{}, builder.WithPredicates(jobUpdatesOnly())).
```

Comment:
Maybe rename this to match the pattern of the other predicates

```suggestion
		Owns(&batchv1.Job{}, builder.WithPredicates(jobUpdatedPredicate())).
```